ipam/api.go

```diff
@@ -29,7 +29,8 @@ type Ipam interface {
 
 // CapabilitiesResponse returns whether or not this IPAM required pre-made MAC
 type CapabilitiesResponse struct {
-	RequiresMACAddress bool
+	RequiresMACAddress    bool
+	RequiresRequestReplay bool
```

Comment:
These are currently documented as is ([see here](https://github.com/moby/moby/blob/606519a1715cc4152c26775aa06f93a6bd3bd4c0/daemon/libnetwork/ipamapi/contract.go#L100-L107)):

```go
type Capability struct {
	// Whether on address request, libnetwork must
	// specify the endpoint MAC address
	RequiresMACAddress bool
	// Whether of daemon start, libnetwork must replay the pool
	// request and the address request for current local networks
	RequiresRequestReplay bool
}
```

Would you mind duplicating these comments such that plugin developers know what they do?